docs/how-to/unified_memory.rst

```diff
@@ -0,0 +1,595 @@
+.. meta::
+  :description: This chapter describes introduces Unified Memory (UM) and shows
+                how to use it in AMD HIP.
+  :keywords: AMD, ROCm, HIP, CUDA, unified memory, unified, memory, UM, APU
+
+*******************************************************************************
+Unified Memory
+*******************************************************************************
+
+Concept
+=======
+
+In a conventional architectures, both CPUs a GPUs have a dedicated memory,
```

Comment:
Recommended text:

Introduction
In conventional architectures, CPUs and GPUs have dedicated memory like Random Access Memory (RAM) and Video Random Access Memory (VRAM). This architectural design, while effective, can be limiting in terms of memory capacity and bandwidth, as continuous memory copying is required to allow the processors to access the appropriate data. New architectural features like Heterogeneous System Architectures (HSA) and Unified Memory (UM) help avoid these limitations and promise increased efficiency and innovation.

Unified memory
Unified Memory can be understood as a single memory address space accessible from any processor within a system. This setup simplifies memory management processes and enables applications to allocate data that can be read or written by code running on either CPUs or GPUs. The Unified memory model is shown in the following figure.

Reply:
Updated.